pkg/errors/errors.go

```diff
@@ -141,6 +142,8 @@ func GetErrorCode(err error) string {
 		return "QuotaViolation"
 	} else if IsMeasurementUnitError(err) {
 		return "MeasurementUnitError"
+	} else if IPOutOfRange(err) {
```

Comment:
This needs to be fixed to the new function name

Reply:
fixed